docs/content/users/extend/additional-services.md

```diff
@@ -58,6 +60,7 @@ Officially-supported add-ons:
 * [Memcached](https://github.com/drud/ddev-memcached): `ddev get drud/ddev-memcached`.
 * [MongoDB](https://github.com/drud/ddev-mongo): `ddev get drud/ddev-mongo`.
 * [PDFreactor](https://github.com/drud/ddev-pdfreactor): `ddev get drud/ddev-pdfreactor`
+* [Platformsh](https://github.com/drud/ddev-platformsh): `ddev get drud/ddev-platformsh`
```

Comment:
Add a period since it’s the product name:

```suggestion
* [Platform.sh](https://github.com/drud/ddev-platformsh): `ddev get drud/ddev-platformsh`
```